actions/class.RdfController.php

```diff
@@ -737,6 +741,80 @@ public function copyInstance()
         ], 412);
     }
 
+    /**
+     * @requiresRight uri READ
```

Comment:
Maybe it is more clarifying `classUri`?

```suggestion
     * @requiresRight classUri READ
```